### DISCUSSION TITLE
Larger margin fune-tuning with ECAPC-TDNN in VoxCeleb recipe

### DISCUSSION THREAD

**Discussion by lhanzl** [speechbrain/speechbrain#2151] [Q&A]
Many papers describe that larger margin tuning can reduce the speaker's performance by at least 10%.
I referred to this paper (https://arxiv.org/abs/2010.11255) for fine-tuning (the pre-training code referred to CommonLanguage), but I found that the Loss was relatively high at the beginning, trained 20 epochs, and the performance did not exceed the basic model, who can give me some suggestions, thank you in advance.

**Comment by mravanelli** [comment]
This looks a question for @Jenthe!

**Comment by lhanzl** [comment]
@Jenthe @mravanelli Thank in advance. I could replay the experiment as you suggested and give you some results.

**Comment by llearner** [comment]
> @Jenthe @mravanelli Thank in advance. I could replay the experiment as you suggested and give you some results.

Hi, do you have some new results ?
